src/azure-cli/azure/cli/command_modules/rdbms/validators.py

```diff
@@ -71,3 +93,24 @@ def validate_subnet(cmd, namespace):
     else:
         raise CLIError('incorrect usage: [--subnet ID | --subnet NAME --vnet-name NAME]')
     delattr(namespace, 'vnet_name')
+
+
+def validate_private_endpoint_connection_id(cmd, ns):
+    connection_id = ns.connection_id
+    connection_name = ns.private_endpoint_connection_name
+    server_name = ns.server_name
+
+    if not connection_id:
+        if not all([connection_name, server_name]):
+            raise argparse.ArgumentError(
```

Comment:
please raise `CLIError('usage error: --name/-n and --server-name')

Reply:
fixed